CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
>I was still wondering about this PR. IMO should not have merged so fast.

sorry! I had assumed it was a minor detail. But we can undo it easily if a different solution is found.